src/utils/test/axe.ts

```diff
@@ -4,7 +4,7 @@ import { axe } from 'jest-axe';
 
 export const axeTest = (
   element: ReactElement<any>,
-  options: any = {},
+  options: any = { rules: { region: { enabled: false } } },
```

Comment:
This needs an update to merge defaults with provided props as default in conflict case.

Reply:
Rules are now shallow merged using provided values as primary and defaults as secondary.